src/util.h

```diff
@@ -9,9 +9,18 @@
 #include <glib.h>
 #include <gio/gio.h>
 #include <tss2/tss2_tpm2_types.h>
+#if defined(__FreeBSD__)
+#include <sys/endian.h>
```

Comment:
Nothing in this module uses `endian.h`.

Reply:
As I said, three *.c files use endian functions and they all include src/util.h. I would need to make the same #if defined(__FreeBSD__) in all three *.c files or in one common header file that is included by all *.c files.

Reply:
Apologies for making your repeat yourself. Without a commit message to explain this though, I don't know of any other way to reverse engineer your intent. How about you fill in the associated commit message with data from this thread when you update the `poll` bits of the patch?

Reply:
PR updated